<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetup</class>
 <widget class="QDialog" name="DialogSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>151</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>151</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>151</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>日志信息过滤设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_debugLog">
          <property name="text">
           <string>正反演信息</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_DPCGRCLog">
          <property name="text">
           <string>DPCGRC收敛信息</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_NroLog">
          <property name="text">
           <string>NRO信息</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>日志基本设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_isWriteFile">
          <property name="text">
           <string>日志是否写入到文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogSetup</receiver>
   <slot>onOkButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOkButton()</slot>
 </slots>
</ui>
